### PR TITLE
feat(Card): Add outline shadow variant

### PR DESCRIPTION
**Pull request by zippyzow**
### Overview

We need to add this [“multi” layer card look on path cards](https://www.figma.com/file/rGzb9m1sPiFihBX3u8WEmG/Catalog-revamp?node-id=828%3A46841) that then transitions on hover. [This is how the hover effect should work](https://www.figma.com/proto/rGzb9m1sPiFihBX3u8WEmG/Catalog-revamp?page-id=801%3A42740&node-id=867%3A49918&viewport=236%2C114%2C0.31&scaling=min-zoom) (ignore text animation for now).

Approved by Tamar and Mark ✅

### PR Checklist

- [x] Related to designs: https://www.figma.com/file/rGzb9m1sPiFihBX3u8WEmG/Catalog-revamp?node-id=828%3A46841
- [x] Related to JIRA ticket: [DISC-921]
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change
- [x] This PR includes testing instructions tests for the code change
- [x] The alpha package of this PR is passing end-to-end tests in all relevant Codecademy repositories

#### Testing Instructions

1. Visit https://63601dfd14a48b1de22e9d1c--gamut-preview.netlify.app/?path=/docs/atoms-card--card#shadow-variants
2. See new shadow variant section with new outline variant 
3. Hover over to see effect
4. Visit mono pr env /catalog page to see effect on career cards ([must be whitelisted in experiment](https://app.optimizely.com/v2/projects/21056540090/experiments/22405500170/whitelist))

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](https://github.com/codecademy-engineering/Codecademy/pull/33301)  | This variant will only be on portal pages |
| Monorepo       | [Portal Link](https://github.com/codecademy-engineering/mono/pull/511)  | [Portal Env](https://tengu.codecademy.com/catalog?PR_ENV=portal-app-pr-511)   |

[DISC-921]: https://codecademy.atlassian.net/browse/DISC-921?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://63601dfd14a48b1de22e9d1c--gamut-preview.netlify.app>https://63601dfd14a48b1de22e9d1c--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/63601dfd14a48b1de22e9d1c>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@7.9.1-alpha.94f597.0<br/>@codecademy/gamut-illustrations@0.32.3-alpha.94f597.0<br/>@codecademy/gamut-kit@0.6.221-alpha.94f597.0<br/>@codecademy/gamut-labs@33.6.12-alpha.94f597.0<br/>@codecademy/gamut-patterns@0.8.16-alpha.94f597.0<br/>@codecademy/gamut-styles@14.1.2-alpha.94f597.0<br/>@codecademy/gamut-tests@4.0.15-alpha.94f597.0<br/>@codecademy/gamut@51.1.2-alpha.94f597.0<br/>@codecademy/markdown-overrides@0.9.13-alpha.94f597.0<br/>@codecademy/monaco@0.6.99-alpha.94f597.0<br/>@codecademy/styleguide@62.0.6-alpha.94f597.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://636025a36c10bc22753a001e--gamut-preview.netlify.app>https://636025a36c10bc22753a001e--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/636025a36c10bc22753a001e>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.222-alpha.5c6e46.0<br/>@codecademy/gamut-labs@33.6.13-alpha.5c6e46.0<br/>@codecademy/gamut@51.1.3-alpha.5c6e46.0<br/>@codecademy/monaco@0.6.100-alpha.5c6e46.0<br/>@codecademy/styleguide@62.0.7-alpha.5c6e46.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://63613d4c555d7400ba525737--gamut-preview.netlify.app>https://63613d4c555d7400ba525737--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/63613d4c555d7400ba525737>Deploy Logs</a></p>

File: packages/gamut/src/Card/index.tsx [hunk 1]
```diff
@@ -5,6 +5,14 @@ import React, { ComponentProps } from 'react';
 
 import { Box } from '../Box';
 
+const outlineStyles = {
+  boxShadow: '-5px 5px white, -5px 5px 0 1px black',
```
Comment (dreamwasp) [inline packages/gamut/src/Card/index.tsx:9]: should these colors change with ColorMode context? we still probably want to pull them from the theme - you can do this with string interpolation like:

```
  boxShadow: `-5px 5px ${theme.colors.white}, -5px 5px 0 1px ${theme.colors.black}`
```

if you do want them to change with ColorMode context, you can sub those in for:

```
  boxShadow: `-5px 5px ${theme.colors.background}, -5px 5px 0 1px ${theme.colors.text}`
```

**Review by dreamwasp — COMMENTED** [review]
have the smallest comment but this is looking great!

**Review by saghdaey — APPROVED** [review]
LGTM! thanks for adding to story book as well

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.222-alpha.8e904e.0<br/>@codecademy/gamut-labs@33.6.13-alpha.8e904e.0<br/>@codecademy/gamut@51.1.3-alpha.8e904e.0<br/>@codecademy/monaco@0.6.100-alpha.8e904e.0<br/>@codecademy/styleguide@62.0.7-alpha.8e904e.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://63618f1fad9d8855754f5ec0--gamut-preview.netlify.app>https://63618f1fad9d8855754f5ec0--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/63618f1fad9d8855754f5ec0>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.224-alpha.df5795.0<br/>@codecademy/gamut-labs@33.6.15-alpha.df5795.0<br/>@codecademy/gamut@51.2.2-alpha.df5795.0<br/>@codecademy/monaco@0.6.102-alpha.df5795.0<br/>@codecademy/styleguide@62.1.2-alpha.df5795.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://636191aa5bb962591c171dda--gamut-preview.netlify.app>https://636191aa5bb962591c171dda--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/636191aa5bb962591c171dda>Deploy Logs</a></p>

**Review by dreamwasp — APPROVED** [review]

**Pull request merged**